openshift_metrics/invoice.py

```diff
@@ -97,9 +86,12 @@ def get_service_unit(self) -> ServiceUnit:
         else:
             return ServiceUnit(SU_UNKNOWN_GPU, 0, "GPU")
 
-        cpu_multiplier = self.cpu_request / su_config[su_type]["cpu"]
-        gpu_multiplier = self.gpu_request / su_config[su_type]["gpu"]
-        memory_multiplier = self.memory_request / su_config[su_type]["ram"]
+        cpu_multiplier = self.cpu_request / int(su_definitions[su_type]["vCPUs"])
+        memory_multiplier = self.memory_request / int((int(su_definitions[su_type]["RAM"])/1024))
+        if int(su_definitions[su_type]["vGPUs"]) != 0:
```

Comment:
If you'd like, I think this is slightly prettier:
```suggestion python
gpu_definition_int = int(su_definitions[su_type]["vGPUs"])
gpu_multiplier = gpu_definition_int or self.gpu_request / gpu_definition_int
```